vowpalwabbit/allreduce_sockets.cc

```diff
@@ -339,7 +339,7 @@ void AllReduceSockets::broadcast(char* buffer, const size_t n)
       int read_size = recv(socks.parent, buffer + parent_read_pos, (int)count, 0);
       if (read_size == -1)
       {
-        THROW(" recv from parent: " << strerror(errno));
+        THROW(" recv from parent: " << VW::strerror_to_string(errno));
```

Comment:
nit: since we're here, is there a purpose on that first space? feel free to ignore this minimal